### PR TITLE
Fix blank transition builder

### PR DESCRIPTION
**Pull request by crisdut**
**Description**

This PR intent fix a blank transition builder when contract has another `ifaceimpl`. This particular case occurs when issuer issues many contracts with different `ifaceimpl `in the same UTXO.

CC @cryptoquick @dr-orlovsky

### CODE REVIEW

File: std/src/stl/specs.rs [hunk 1]
```diff
@@ -477,8 +477,9 @@ impl DivisibleAssetSpec {
     pub fn details(&self) -> Option<&str> { self.naming.details.as_ref().map(|d| d.as_str()) }
 }
 
-#[derive(Clone, Eq, PartialEq, Debug, Default)]
-#[derive(StrictType, StrictEncode, StrictDecode)]
+#[derive(Wrapper, Clone, Ord, PartialOrd, Eq, PartialEq, Hash, From, Debug)]
```
[the remark this answers is not in the document]
Comment (crisdut) [reply std/src/stl/specs.rs:480]: `Default` causes conflict with `strict_dumb`. Do you prefer maintain Default?

File: std/src/stl/specs.rs [hunk 2]
```diff
@@ -498,6 +499,13 @@ impl FromStr for RicardianContract {
     }
 }
 
+impl RicardianContract {
+    pub fn from_strict_val_unchecked(value: &StrictVal) -> Self {
+        let s = value.unwrap_string();
+        RicardianContract::from_str(&s).expect("invalid term data")
```
[the remark this answers is not in the document]
Comment (crisdut) [reply std/src/stl/specs.rs:505]: I didn't know, as I saw in several code snippets, I imagined that this was the correct implementation. What would be the alternative?

File: std/src/stl/specs.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (dr-orlovsky) [reply std/src/stl/specs.rs:480]: I think having default is better

File: std/src/stl/specs.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (dr-orlovsky) [reply std/src/stl/specs.rs:505]: Why do you need an unchecked version?

[the remark this answers is not in the document]
Comment (crisdut) [reply std/src/stl/specs.rs:505]: I have a method that extracts some information from the `ContractIface::global` method, and this method returns the StrictVal array. I use **from_strict_val_unchecked** to convert StrictVal to a specific structure.

[the remark this answers is not in the document]
Comment (dr-orlovsky) [reply std/src/stl/specs.rs:505]: Can you point specific place in code where you do that call?

[the remark this answers is not in the document]
Comment (crisdut) [reply std/src/stl/specs.rs:505]: Hi,

I used here: https://github.com/diba-io/bitmask-core/blob/development/src/rgb/contract.rs#L199-L203

[the remark this answers is not in the document]
Comment (dr-orlovsky) [reply std/src/stl/specs.rs:505]: Why you do not want to use `try_from` and just propagate the error?

**Comment by dr-orlovsky** [comment]
Can you please rebase on the master to reflect everything which has happened in v0.11 branch?

**Pull request closed without merging**